README.md

```diff
@@ -160,6 +192,7 @@ by Elliott Back. The author has applied the following further changes:
 - Changed the API to take string slices (pointer & length) instead of null
   terminated strings.
 - Did a pass to clean up the comments and function signatures.
+- Added second iterator, tests and documentation by Samuel D. Crow
```

Comment:
Minor minor nit thing - but can you put a period '.' at the end of the sentence?

Reply:
It's a sentence fragment but I can add one if you want one.

Reply:
Aye just cause the rest are non-fragments and I'm completely anal about consistency :D

Reply:
Done!